Gems/AWSClientAuth/Code/Include/Authentication/AuthenticationProviderBus.h

```diff
@@ -29,7 +29,8 @@ namespace AWSClientAuth
         //! @return bool True if valid access token available, else False
         virtual bool IsSignedIn(const ProviderNameEnum& providerName) = 0;
 
-        //! Get cached tokens from last last successful sign-in for the provider.
+        //! [Deprecated] Get cached tokens from last last successful sign-in for the provider.
+        //! The returned access tokens will only include the refresh token.
```

Comment:
by changing this to only returning the `refresh` token, will this API no longer serve its intended purpose? What's the impact on anyone who might be calling this now?

Reply:
Yeah, this API should be deprecated soon since we no longer cache the access token and id token in the memory. However, these tokens will still be broadcasted with the SignInSuccess notifications. Developers are expected to handle those notifications in their project code to get the tokens and store them by themselves instead of retrieving a copy from the Client Auth gem. I can update the API description to include the suggested solution.